src/Microsoft.DotNet.Interactive/Microsoft.DotNet.Interactive.Netstandard20.csproj

```diff
@@ -0,0 +1,12 @@
+<Project>
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <PackageId>Microsoft.DotNet.Interactive.Netstandard20</PackageId>
+    <PackageDescription>Core types for building applications providing interactive programming for .NET. for the desktop</PackageDescription>
+    <PackageTags>interactive desktop</PackageTags>
+  </PropertyGroup>
+
+  <Import Project="Microsoft.DotNet.Interactive.csproj" />
```

Comment:
If you do the `<Import>` before the `<PropertyGroup>` then you can remove a bunch of the parts that look like this `Condition="$(PackageId) == ''"` because they'll just get overwritten.